includes/gateways/stripe/class-give-stripe.php

```diff
@@ -59,6 +59,21 @@ public function __construct() {
 		 */
 		public function includes() {
 
+			// Include admin files.
+			require_once GIVE_PLUGIN_DIR . 'includes/gateways/stripe/includes/admin/admin-helpers.php';
+			require_once GIVE_PLUGIN_DIR . 'includes/gateways/stripe/includes/admin/admin-actions.php';
+			require_once GIVE_PLUGIN_DIR . 'includes/gateways/stripe/includes/admin/admin-filters.php';
+			require_once GIVE_PLUGIN_DIR . 'includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php';
```

Comment:
@mehul0810 Can you start using PHPStorm because it keeps track of useful code which we can remove.

As you can see in below image `$give_stirpe` is not used in `__contruct`, so safe to remove.
![image](https://user-images.githubusercontent.com/1784821/63491781-bb3e2100-c4d5-11e9-8b2b-11911ad941d2.png)

Reply:
I've started using PHPStorm, Fixed!